sdk/servicebus/Azure.Messaging.ServiceBus/api/Azure.Messaging.ServiceBus.netstandard2.0.cs

```diff
@@ -32,6 +32,7 @@ public partial class ProcessMessageEventArgs : System.EventArgs
         public string EntityPath { get { throw null; } }
         public string FullyQualifiedNamespace { get { throw null; } }
         public string Identifier { get { throw null; } }
+        public System.Threading.CancellationToken LockLostCancellationToken { get { throw null; } }
```

Comment:
What about calling this MessageLockCancellationToken?

Reply:
Whatever you prefer ;) Just let me know

For me given the event args already contain `Message` in `ProcessMessageEventArgs` it feels slightly redundant but I do not have skin in the game to argue ;)

Reply:
My comment is more about whether we should have Lost in the name, but just removing "Lost" ends up with LockCancellationToken, which sounds odd. This is the train of thought that led me to MessageLockCancellationToken. We will discuss this more with the .NET architects as well.

Reply:
Is it because `LockLost` is hard to read? How about `LockExpiredCancellationToken`? I was thinking `LockLost` comes closes to the actual service reason and is domain specific vocabulary that is already in place

Reply:
LockExpiredCancellationToken?

Reply:
Sorry my comment wasn't about the specific word "Lock" but rather that we are tying the name to the specific event that would result in the token being cancelled. In my mind, it makes more sense for the name to correspond to the concept that the token is associated with, rather than the specific event/point in time in which the token would be cancelled. The token is associated with the message lock. When the lock is lost, the token is canceled. That said, I don't feel super strongly about this way of thinking about it, and I can certainly see the argument for being very explicit.

Reply:
I get that reasoning. I think it is pretty sound. As another example a cancellation token that is related to stopping the endpoint would probably also be called `StopCancellationToken` representing the phase of stopping and not `StoppingCancellationToken` or `StoppedCancellationToken`.

Reply:
I will push a proposal. We can always adjust